MainGame.py

```diff
@@ -15,31 +19,53 @@ def PrepareGame(self) -> None:
         self.playerTwo.name = self.GetUserName("Spieler 2")
         self.playerTwo.symbol = "O"
 
-    def Turn(self, player: Player) -> None:
+    def Turn(self, player: Player) -> Optional[Player]:
```

Comment:
```suggestion
    def Turn(self, player: Player) -> Player:
```

The method always returns a `Player`